catalogue/webapp/pages/search/images.tsx

```diff
@@ -56,6 +56,8 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
   query,
 }): ReactElement<Props> => {
   const { query: queryString } = query;
+  const [isEnhanced, setIsEnhanced] = useState(false);
+  useEffect(() => setIsEnhanced(true));
```

Comment:
You can fetch `isEnhanced` from `AppContext`

Reply:
solved